apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/DefaultDataSetMonitorListener.java

```diff
@@ -47,28 +52,33 @@ public final class DefaultDataSetMonitorListener implements DataSetMonitorListen
 
   @Override
   public void dataAvailable(RifFilesEvent rifFilesEvent) throws Exception {
-    Timer.Context timerDataSet =
-        appMetrics
-            .timer(
-                MetricRegistry.name(
-                    PipelineApplication.class.getSimpleName(), "dataSet", "processed"))
-            .time();
+    Timer.Context timerDataSet = appMetrics.timer(TIMER_PROCESSING).time();
 
     Exception failure = null;
     for (RifFileEvent rifFileEvent : rifFilesEvent.getFileEvents()) {
+      final RifFile rifFile = rifFileEvent.getFile();
+      if (!rifFile.requiresProcessing()) {
+        LOGGER.info("Skipping previously processed file {}", rifFile.getDisplayName());
```

Comment:
Maybe a debug? I imagine the log would be noisy with this line eventually.

Reply:
This should only happen following an interrupt and resume so I felt it was significant enough to make an INFO.  If it happened frequently I'd agree with making it a DEBUG.